CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -551,7 +551,7 @@ public let EXPR_NODES: [Node] = [
     children: [
       Child(
         name: "Identifier",
-        kind: .token(choices: [.token(tokenKind: "IdentifierToken")])
+        kind: .token(choices: [.token(tokenKind: "IdentifierToken"), .keyword(text: "self"), .keyword(text: "Self"), .keyword(text: "init")])
```

Comment:
Editor placeholder is always <##> so ... seems weird we'd need this.

Reply:
You’re right, we don’t actually need it here.